test/ConstructiveSolidGeometry/CSG_primitives.jl

```diff
@@ -161,4 +161,58 @@ no_translations = (rotation = one(SMatrix{3, 3, T, 9}), translation = zero(Carte
         @test !in(CartesianPoint{Float64}(1,1,3+tol),box_closed_trafo)
         @test in(CartesianPoint{Float64}(1,1,3-tol),box_open_trafo)
     end
+    @testset "RegularPrism" begin
+        prism1 = CSG.RegularPrism(CSG.ClosedPrimitive,r=1f0, hZ = 2f0)
+        prism2 = CSG.RegularPrism{Float32}(r=1.0, hZ = 2f0)
+        @test prism1 === prism2
+        
+        dict = Dict("TriangularPrism"   => Dict(
+                "r"       => 1.0,
+                "h"        => 1.0))
+        prism = Geometry(T,dict,default_units,no_translations)
+        output = Dictionary(prism)
+        @test dict == output
+        
+        dict = Dict("QuadranglePrism"   => Dict(
+                "r"       => 1.0,
+                "z"      =>1.0))
+        prism = Geometry(T,dict,default_units,no_translations)
+        output = Dictionary(prism)
+        output["QuadranglePrism"]["z"]=output["QuadranglePrism"]["h"]#Workaround for tests, since dictionary uses the key "h", since "z" is outdated
+        delete!(output["QuadranglePrism"], "h")
+        @test dict == output
+        
+        dict = Dict("PentagonalPrism"   => Dict(
+                "r"       => 1.0,
+                "z"      =>(1.0,2.0)))
```

Comment:
`Geometry` is usually only used to parse dictionaries from a config file. There, one can write
```yaml
z : [1.0, 2.0]
```
where `z` should be parsed an an `Array` rather than a `Tuple`. 
So, if the user specifies `z: (1.0, 2.0)`, then julia parses `z` as `Tuple`?

Reply:
````
if z isa Real
    @warn "Deprecation warning: Field `z` for `RegularPrism` is deprecated. 
        Use `h` instead to specify the height of the primitive."
    z
else # z isa Tuple
    @warn "Deprecation warning: Field `z` for `RegularPrism` is deprecated. 
        Use `h` instead to specify the height of the primitive.
        There might be a conflict with the possible field `origin`:
        The `z` component of the origin of the primitive is overwritten by the `z`."
    origin = CartesianPoint{T}(origin[1], origin[2], mean(z))
    (z[2] - z[1])/2
end 
````

Here we kind of allow a tuple with the comment (code is part of Geometry method)

Reply:
Already in this pull request?

Reply:
No, please just leave it as it is and make a dedicated PR later where you remove the deprecated support for `z`. Also for other primitives.